src/assets/icons/burger.svg

```diff
@@ -0,0 +1,19 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="40" height="27" viewBox="0 0 40 27">
```

Comment:
you need to cooperate with that guy - https://github.com/akai-org/polifood-client/pull/27 😉